### ISSUE TITLE
Missing Keyboard focus for Picker and Lists

### ISSUE THREAD

**Issue by Harbs** [adobe/spectrum-css#926]
## Description
In the React components there is a blue line at the left of a list item which has keyboard foucs.

## Steps to reproduce

1. Go to https://react-spectrum.adobe.com/react-spectrum/ListBox.html or https://react-spectrum.adobe.com/react-spectrum/Picker.html
2. Click on a list and press up and down arrow keys.
3. Observe a blue indicator on the list item which has focus.

## Expected behavior
There should be css for that inidicator

**Comment by GarthDB** [comment]
Good point. We'll have to look into the custom css they added to fix it. I'm not sure we can get the up/down arrows to work without JS.

**Comment by GarthDB** [comment]
@Harbs I'm seeing that blue line for keyboard focus on the dropdown component 
![image](https://user-images.githubusercontent.com/125516/93919061-7719e300-fcca-11ea-87f0-cf49bc4f7052.png)
And Menu:
![image](https://user-images.githubusercontent.com/125516/93919163-8ac54980-fcca-11ea-9aed-3ceb4bd73211.png)

Where are you seeing the issue of it not being there?

**Comment by Harbs** [comment]
Where are those screenshots from?

I see them in the React components, but I don't see them anywhere in the spectrum-css examples. I thought they might just be missing from the documentation, but looking through the source CSS, I could not find CSS definitions for that.

**Comment by Harbs** [comment]
Ah. OK I do see the css in menu/skin.css.

I see it's supposed to be implemented using `focus-ring`. The same goes for many other components such as `AssetList` etc. It's probably a good idea adding this to the documentation.

I have what I need to implement this.

Thanks! 😄
